build-logic/src/main/kotlin/fr/androidmakers/gradle/android/application/AndroidApplicationPlugin.kt

```diff
@@ -22,34 +22,18 @@ class AndroidApplicationPlugin : Plugin<Project> {
     project.apply(mapOf("plugin" to "com.google.gms.google-services"))
 
     addRepositories(project)
-    checkGoogleServices(project)
 
     project.androidSetup()
 
     project.extensions.findByName("android")!!.apply {
       this as BaseExtension
       defaultConfig.apply {
-        applicationId = "fr.paug.androidmakers"
+        namespace = "fr.paug.androidmakers"
```

Comment:
I wanted to give a shot at the namespace thing, which took me down the rabbit hole of updating to the latest AS canary because Bumblebee doesn't find the application to start if `package` is not in the Manifest
. It's all working with Dolphin!